src/components/config/config.ts

```diff
@@ -262,3 +262,8 @@ export function isContextStatusBarDisabled(): boolean {
 export function isNamespaceStatusBarDisabled(): boolean {
     return vscode.workspace.getConfiguration(EXTENSION_CONFIG_KEY)['vs-kubernetes.disable-namespace-info-status-bar'];
 }
+
+// gets the default local tunnel debug provider to use
```

Comment:
Nit: Is this actually the `default`? Should it be more the `debug provider defined in settings` or something of that kind?

Reply:
Yeah, it's the one defined in settings. Will update